README.md

```diff
@@ -68,22 +68,22 @@ list s3://linz-imagery/sample --include ".*.tiff$" --group 10 --group-size 100MB
 list s3://linz-imagery/sample --include ".*.tiff$"  --exclude "BG33.tiff$" --output /tmp/list.json
 ```
 
-### `pretty-print`
+### `format-json`
 
-Format all JSON files within a directory using `prettier`.
+Format all JSON files within a directory using `prettier`. An optional flag `--fix-content-type` allows to set the content-type for s3 object with a value depending on the STAC object type.
 
 #### Example
 
 - Format and overwrite files:
 
 ```bash
-pretty-print source/
+format-json source/ [--fix-content-type]
```

Comment:
How come the name change? prettier will format a lot more than json. eg markdown, yaml, javascript etc..

Reply:
Because it's looking for `json` files atm. I can make the function more generic and look at other file format but I feel it will need a bit more work, specially around the `--fix-content-type`

Reply:
This is a breaking change to rename it, so Id prefer to leave it as is if we can?